src/pages/data-entry/data-entry.jsx

```diff
@@ -1,127 +1,214 @@
-import React, { Component } from "react";
+import React, { Component } from 'react'

```

Comment:
Need to make mutations and queries in a way where you can detect and adjust the view for cases where the request fails (and it can fail (our blokchain or server could be down, user may not have internet connectoin, etc)). Now that you have more context on how this works, review this:
https://www.apollographql.com/docs/react/get-started/#request-data